api/base/serializers.py

```diff
@@ -143,92 +173,141 @@ def get_attribute(self, obj):
         return field_source_method(auth=auth)
 
 
-class JSONAPIHyperlinkedIdentityField(ser.HyperlinkedIdentityField):
+class RelationshipField(ser.HyperlinkedIdentityField):
     """
-    HyperlinkedIdentityField that returns a nested dict with url,
-    optional meta information, and link_type.
+    RelationshipField that permits the return of both self and related links, along with optional
+    meta information.
 
     Example:
+    children = RelationshipField(
+        related_view='nodes:node-children',
+        related_view_kwargs={'node_id': 'pk'},
+        self_view='nodes:node-node-children-relationship',
+        self_view_kwargs={'node_id': 'pk'},
+        related_meta={'count': 'get_node_count'}
+    )
 
-        children = JSONAPIHyperlinkedIdentityField(view_name='nodes:node-children', lookup_field='pk',
-                                    link_type='related', lookup_url_kwarg='node_id', meta={'count': 'get_node_count'})
+    If you surround the lookup_field in angular brackets, it will not attempt to find that attribute on the target, but rather
```

Comment:
I disagree that it is more intuitive - to me, it reads as an exception.  HyperlinkedIdentityField and HyperlinkedRelatedField don't use angular brackets.  I don't know if matching ListField is a good reason to do it - we rarely use a Link with kwargs and have replaced the field in many places.  I think it would make more sense to change ListField to get rid of the angular brackets.